### PR TITLE
Fix doc build

### PR DESCRIPTION
**Pull request by davimacedo**

### CODE REVIEW

File: src/OfflineQuery.js [hunk 1]
```diff
@@ -83,11 +83,11 @@ function equalObjectsGeneric(obj, compareTo, eqlFn) {
 /**
  * Converts human readable relative date string
  * ie: 'in 10 days' to a date relative to now
+ * copied from parse-server/src/Adapters/Storage/Mongo/MongoTransform.js
  *
  * @param {*} text
  * @param {*} now
- * @returns {status,info,result}
- * copied from parse-server/src/Adapters/Storage/Mongo/MongoTransform.js
+ * @returns {{status:string,info:string,result:Date}}
```
Comment (mtrezza) [inline src/OfflineQuery.js:90]: The docs only say return type is `Object`, without any more details:

image

Is that what you intended? Or should we add a @typedef to document that output object in detail?

Comment (mtrezza) [reply src/OfflineQuery.js]: I added a @typedef to complete the docs, is that ok?

image

**Comment by davimacedo** [comment]
Yes. It looks good to me. It's important to merge this one because the docs failed to build on last release.

**Review by mtrezza — APPROVED** [review]

**Comment by mtrezza** [comment]
Yes, sorry for the long wait time, I noticed earlier that I only subscribed to releases of this repo, so I never received any notifications for review.

**Pull request merged**